node-notes-cli/app.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+
+const keyWord = process.argv[2];
+const keyWordLib = ['read', 'create', 'update', 'delete'];
+
+if (!keyWordLib.includes(keyWord)) {
+  console.error({ error: 'invalid command' });
+} else if (keyWordLib.includes(keyWord)) {
+  if (keyWord === keyWordLib[0]) {
+    fs.readFile('data.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error(err);
+      }
+      const notesObj = JSON.parse(data);
+      for (const notesId in notesObj.notes) {
+        console.log(notesId + ': ' + notesObj.notes[notesId]);
+      }
+    });
+  } else if (keyWord === keyWordLib[1]) {
+    fs.readFile('data.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error(err);
+      }
+      const notesObj = JSON.parse(data);
+      const newId = notesObj.nextId;
+      if (!process.argv[3]) {
+        console.error({ error: 'invalid argument. please input a string' });
+        return;
+      }
+      notesObj.notes[newId] = process.argv[3];
+      notesObj.nextId++;
+      console.log(`Note ID ${newId} successfully created`);
+      fs.writeFile('data.json', JSON.stringify(notesObj, null, 2), 'utf8', err => {
+        if (err) {
+          console.error(err);
+        }
+      });
+    });
+  } else if (keyWord === keyWordLib[2]) {
+    const inputId = process.argv[3];
+    fs.readFile('data.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error(err);
+      }
+      const notesObj = JSON.parse(data);
+      if (isNaN(Number(inputId)) || Math.floor(inputId) !== Number(inputId)) {
+        console.error({ error: 'id must be a whole number' });
+        return;
+      } else if (!notesObj.notes[inputId]) {
+        console.error({ error: 'id does not exist' });
+        return;
+      } else if (!process.argv[4]) {
+        console.error({ error: 'invalid argument. please input a string' });
+        return;
+      }
+      for (const notesObjId in notesObj.notes) {
+        if (notesObjId === inputId) {
+          notesObj.notes[notesObjId] = process.argv[4];
+          console.log(`Note ID ${notesObjId} successfully updated`);
+        }
+      }
+      fs.writeFile('data.json', JSON.stringify(notesObj, null, 2), 'utf8', err => {
+        if (err) {
+          console.error(err);
+        }
+      });
+    });
+  } else if (keyWord === keyWordLib[3]) {
```

Comment:
Please move the names of the operations down here from the hard-coded array at the top so it's easier to tell which operation each block of code is attempting to implement.